flash/text/question_answering/data.py

```diff
@@ -202,8 +388,14 @@ def from_squad_v2(
         doc_stride: int = 128,
         **data_module_kwargs: Any,
     ) -> "QuestionAnsweringData":
-        """Creates a :class:`~flash.text.question_answering.data.QuestionAnsweringData` object from the given data
-        JSON files in the SQuAD2.0 format.
+        """Load the :class:`~flash.text.question_answering.data.QuestionAnsweringData` from JSON files containing
```

Comment:
Let's eventually have a SQuAD format doctest too. Happy for you to open an issue for this and leave for future work if you like :smiley:

Reply:
I think we can add this with this PR itself, need to properly set the SQuAD format for the dummy data.
I will update the PR with this.

Reply:
Cool, sounds good to me :smiley:

Reply:
I have updated the PR with doctest for `from_squad_v2`.